reader.go

```diff
@@ -131,31 +133,33 @@ func (r *Reader) Next() (ok bool) {
 // Err returns the latest error of a reader.
 func (r *Reader) Err() error { return r.err }
 
-func (r *Reader) Bind(column string, value Value) {
+// Bind binds a column in a csv to its matching Value struct.
+// It's useful when you are interested in adding your own Value types.
+func (r *Reader) Bind(colName string, value Value) {
 	if r.parser != nil {
 		panic("binding must be done before calling Next")
 	}
-	r.columns = append(r.columns, Column{Name: column, Index: IndexUnbound, Value: value})
+	r.colsOfInterest = append(r.colsOfInterest,
+		Field{
+			Name:   colName,
+			Column: ColumnUnbound,
+			Value:  value,
+		},
+	)
 }
 
-func (r *Reader) StringColumn(column string) *String {
+// String returns a pointer to a string field value of a given colName
+// which is reassigned with every Next() call.
+func (r *Reader) String(colName string) *string {
```

Comment:
Maybe `colname`, `columnName` or `column` or just `name`. `colName` somehow feels really weird for me.

Reply:
I renamed it to `columnName`. Addressed in 993aac5